tests/telemetry/telemetry_utils.py

```diff
@@ -101,6 +101,7 @@ def listen_for_events(duthost, gnxi_path, ptfhost, filter_event_regex, op_file,
     cmd = generate_client_cli(duthost=duthost, gnxi_path=gnxi_path, method=METHOD_SUBSCRIBE,
                               submode=SUBMODE_ONCHANGE, update_count=update_count, xpath="all[heartbeat=2]",
                               target="EVENTS", filter_event_regex=filter_event_regex, timeout=timeout)
+    cmd = f"timeout {timeout} " + cmd
```

Comment:
I don't understand this change. Why are we additionally adding an extra timeout? Timeout is already part of gnmi client command.